docs/style-spec/expressions.md

```diff
@@ -0,0 +1,40 @@
+# Style Function Expressions
+
+**Constants:**
+- LN2
+- PI
+- e
```

Comment:
The issue with introducing a mini-syntax inside of string literals is that you then also have to include an escaping syntax, so that the literal string "$pi", or other literals starting with a dollar sign, remain expressible. If you pick backslash as the escape character, then that has to be compounded with the escaping syntax of JSON itself (`"\\$pi"`). This is confusing accidental complexity. Better to keep the grammar simple and use s-expressions as the _only_ syntactical element. Expose constants as _constant functions_: `["pi"]`.